public/app/features/alerting/unified/components/settings/__mocks__/server.ts

```diff
@@ -0,0 +1,25 @@
+import { http, HttpResponse } from 'msw';
```

Comment:
I like this approach, and I think it could be extended/generalised for all of our alerting MSW behaviour (we should continue our previous chat about it :+1:)